### DISCUSSION TITLE
Flow update (validation) summary field concatenatiom

### DISCUSSION THREAD

**Discussion by mhdaouas** [directus/directus#14424] [Archive]
Hi!

What is the easiest way to concatenate the value of a string field to write the summary of workflow validations every time a user updates a form (validates it). I want to add a line with the name of the validator, his decision (validation/rejection that I handle with a dropdown field) and validation datetime.

Thanks!

**Comment by rijkvanzanten** [comment]
You could use the `log` operation, and log something like `Update: {{payload.name}} {{payload.status}} {{payload.timestamp}}`. That being said, you gotta make sure those payload values exist. In the update event, the trigger data only contains the data that was actually changed, so you might need to add a `read` operation to get the fully saved item first 👍🏻

**Reply by mhdaouas** [reply]
Yes but I want to show this in a readonly textarea field in my form. Is it possible?

**Reply by rijkvanzanten** [reply]
You should be able to setup a readonly field in the data model, and then update it's value in the `update` operation in a flow 👍🏻
